super_dpf.py

```diff
@@ -89,8 +92,26 @@ def verify_local(self, cloud_filelist, local_filelist):
                 log.info('Deleting local file {}'.format(local_file))
                 os.remove(local_file)
 
-    def stylize_image(self, image):
-        return image
+    def stylize_image(self, img):
+        try:
+            background = img.copy()
+
+            w, h = img.size
+            ratio = FOREGROUND_SIZE['height'] / h
```

Comment:
These should probably be cast to floats before being divided.  Integer arithmetic will do you no favors when coming up with a ratio factor.

If you want to solve simple proportions while staying in integer territory you will need to multiply before you get to the division step instead of doing the division once and storing it.